SunEngine.Tests/SunEngine.Admin/Managers/CacheSettingsManagerTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using SunEngine.Admin.Managers;
+using SunEngine.Core.Cache.Services;
+using SunEngine.Core.Configuration.Options;
+using SunEngine.Core.DataBase;
+using SunEngine.Core.Models;
+using Xunit;
+
+namespace SunEngine.Tests.SunEngine.Admin.Managers
+{
+    public class CacheSettingsManagerTests
+    {
+        private CacheSettingsManager cacheSettingsManager { get; set; }
+        private DataBaseConnection dbConnection { get; set; }
+
+        public CacheSettingsManagerTests()
+        {
+            dbConnection = DefaultInit.GetTestDataBaseConnection();
+            cacheSettingsManager = InitialiseDefaultSettingsManager();
+        }
+
+        private CacheSettingsManager InitialiseDefaultSettingsManager()
+        {
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddOptions<IOptions<CacheOptions>>();
+            serviceCollection.AddMemoryCache();
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+
+            var categoriesCache = new CategoriesCache(DefaultInit.GetTestDataBaseFactory());
+            var cacheOptions = serviceProvider.GetRequiredService<IOptions<CacheOptions>>();
+            var contentCache = new CategoryContentCache(categoriesCache, serviceProvider.GetRequiredService<IMemoryCache>(), serviceProvider.GetRequiredService<IOptions<CacheOptions>>());
+            return new CacheSettingsManager(cacheOptions, contentCache, dbConnection);
+        }
+
+        [Fact]
+        public async void ShouldUpdateCachePolicy()
+        {
+            using (var transaction = dbConnection.BeginTransaction())
+            {
+                var cacheSettings = dbConnection.CacheSettings.First();
+                var before = cacheSettings.InvalidateCacheTime;
+                cacheSettings.InvalidateCacheTime = 0;
+                await cacheSettingsManager.UpdateCachePolicy(cacheSettings);
+                var after = dbConnection.CacheSettings.First().InvalidateCacheTime;
+                transaction.Rollback();
+                Assert.NotEqual(after, before);
+            }
+        }
+
+        [Fact]
+        public async void ShouldThrowExceptionWhenCachePolicyIsNotDefined()
+        {
+            var cacheSettings = new CacheSettings{ CachePolicy = (CachePolicy)4, Id = 1, InvalidateCacheTime = 10};
+            await Assert.ThrowsAsync<ArgumentOutOfRangeException>(async () =>
+                await cacheSettingsManager.UpdateCachePolicy(cacheSettings));
+        }
+
```

Comment:
Please, leave only one line break.

Reply:
What about readability? I see horizontal scrollbar if use it one line break

Reply:
As for lines 58 and 59 good job there! It's a proper way to format code so we do not have those scroll bars.

As for my point, I meant 61 and 62 line breaks. It's usually not required to have several line breaks after the method.
When you write/read the code, you usually will use hotkeys and searches for fast navigation between methods, so there is no point to separate them or leave additional linebreaks.